doc/src/base/base.md

```diff
@@ -288,6 +288,9 @@ Base.@elapsed
 Base.@allocated
 Base.EnvDict
 Base.ENV
+Base.Sys.cpu_summary
+Base.Sys.cpu_info
+Base.Sys.isapple
```

Comment:
```suggestion
```

Reply:
Sorry, why are we taking `isapple` out (does it not exist anymore?)?

Reply:
It’s already there, just two lines below. :)